1064-2.c

```diff
@@ -0,0 +1,24 @@
+#include <stdio.h>
+
+float num,positivo,media;
+int i;
+int main()
+
+{
+    positivo = 0;

```

Comment:
Você poderia definir a variavel positivo aqui ao inves de deixar ela global (esse comentário vai fazer mais sentido quando você estudar funções).
```suggestion
    int positivo = 0;

```